### PR TITLE
Added section on invite codes to documentation

### PR DESCRIPTION
**Pull request by ankonym**
The webapp configuration section now has a section on enabling and generating invite codes.

### CODE REVIEW

File: pages/docs/platform/services/webapp.md [hunk 1]
```diff
@@ -48,6 +48,25 @@ By putting this in `services/webapp.json`, all the `webapp` nodes will inherit t
...
+      "webapp": {
+        "invite_required": true
+      }
+    }    
+
+Run `leap deploy` to enable the option.
+
+You can then generate invite codes by running 
+    
+  `RAILS_ENV=production bundle exec generate_invites[x,y]`
```
Comment (kaeff) [inline pages/docs/platform/services/webapp.md]: Isn't the line missing the `rake`? (i.e. `RAILS_ENV=production bundle exec **rake** generate_invites[x,y]`

Comment (kaeff) [inline pages/docs/platform/services/webapp.md]: Shall we hint that this needs to be execute from the webapp directory?

Comment (ankonym) [inline pages/docs/platform/services/webapp.md]: Uhm. Oops. Yes! Will add the `rake`... 

and the hint about executing it from the webapp directory. Thanks for catching that!

**Comment by ivanalejandro0** [comment]
You should squash your commits into a single commit.
Otherwise I'd say that whoever merges this should rebase/squash it.

**Pull request merged**